config/environments/development.rb

```diff
@@ -29,7 +29,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :cloudinary
```

Comment:
remember to change this in the `config/environments/production.rb` for this to work on heroku